plugin.py

```diff
@@ -46,5 +47,56 @@ def on_pre_server_command(self, params: Mapping[str, Any], done_callback: Callab
             done_callback()
         return True
 
+    def on_server_response_async(self, method: str, response: Response) -> None:
+        if method == 'codeAction/resolve':
+            self.maybe_handle_move_code_action(response)
 
+    def maybe_handle_move_code_action(self, response: Response) -> None:
+        session = self.weaksession()
+        if not session:
+            return
+        result = cast(MaybeMoveFunctionCodeAction, response.result)
+        command = result.get('command')
+        if not command:
+            return
+        command_name = command.get('arguments')[0]
+        params = command.get('arguments')[1]
+        function_name = command.get('arguments')[2]
```

Comment:
Could simplify with:

```suggestion
        command_name, params, function_name = command.get('arguments')
```

but  I'm not sure you can assume that the array will always have 3 items. Probably should check for that first.

Reply:
it will always have 3 items in the array:
https://github.com/elm-tooling/elm-language-server/blob/171c1399a0327e4381a2052e268037ec5459ca37/src/providers/codeAction/moveFunctionCodeAction.ts#L40-L43

Reply:
We don't know that at this point as we're checking command_name later.